### PR TITLE
Vs code extension testing

### PR DESCRIPTION
**Pull request by sinapsist**

### CODE REVIEW

**Comment by hu3man** [comment]
Is this ready for review @sinapsist ?

**Comment by hu3man** [comment]
@sinapsist, I'm unable to build this locally on my machine

Please advise, thanks!

**Review by mweichert — APPROVED** [review]
Looks good!

**Review by hu3man — APPROVED** [review]
It's working! 

Looks great @sinapsist

File: taqueria-vscode-extension/src/test/suite/data/vscode-taq-test-project/quickstart.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
+## Overview
```
Comment (russellmorton) [inline taqueria-vscode-extension/src/test/suite/data/vscode-taq-test-project/quickstart.md:1]: Should this file be removed?

**Pull request merged**